sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -355,7 +361,25 @@ struct graphs_support;
 } // namespace device
 } // namespace info
 
-class node {};
+template <typename ValueT>
+class dynamic_parameter {
+public:
+  dynamic_parameter(command_graph<graph_state::modifiable> graph);
+
+  dynamic_parameter(const ValueT &initialValue, command_graph<graph_state::modifiable> graph);
+
+  void register(int argIndex, const handler& cgh);
+
+  void update(const ValueT& newValue);
+};
+
+class node {
+public:
+  node() = delete;
+
+  template <int Dimensions>
+  void update_nd_range(nd_range<Dimensions> executionRange);
+};
```

Comment:
It looks like the synopsis for `class node` is duplicated below also.

Reply:
> Note also that the synopsis is missing the new member function `update_range`.

This is now fixed.

I've also removed the duplicate for now based on https://github.com/intel/llvm/pull/12486#discussion_r1498066839 so will resolve this.